src/app/core/entity-components/entity-utils/entity-select/entity-select.component.spec.ts

```diff
@@ -154,29 +152,22 @@ describe("EntitySelectComponent", () => {
     expect(component.selectedEntities).toBeEmpty();
   });
 
-  it("autocompletes with the default accessor", (done) => {
+  it("autocompletes with the default accessor", () => {
     component.allEntities = testUsers;
     component.loading.next(false);
-    let iterations = 0;
     let expectedLength = 4;
-    subscription = component.filteredEntities.subscribe((next) => {
-      iterations++;
-      expect(next.length).toEqual(expectedLength);
-      if (iterations === 4) {
-        done();
-      }
-    });
-    expectedLength = 4;
+
     component.formControl.setValue(null);
     expectedLength = 3;
     component.formControl.setValue("A");
     expectedLength = 3;
     component.formControl.setValue("Ab");
     expectedLength = 1;
     component.formControl.setValue("Abc");
+    expect(component.filteredEntities.length).toEqual(expectedLength);
```

Comment:
This test needs to be adjusted a little bit. `component.filteredEntities.length` should be checked after each call of `component.formControl.setValue(...)` to verify the expected length is correct. At the moment this only makes one check at the end.

Reply:
I've updated this test. Does it look OK now?